### ISSUE TITLE
EventEmitter & TextStyle in pixi v4 (reported in html5gamedevs.com)

### ISSUE THREAD

**Issue by adireddy** [pixijs/pixi-haxe#93]
In Pixi v3 it was impossible to extends EventEmitter because the resource was loaded at runtime

Unless I'm wrong, EventEmitter code is now include in the pixi.js v4 file.

I have tried to extends EventEmitter in v4 but have this error: Uncaught ReferenceError: EventEmitter is not defined

The haxe extern has no reference to a JS Object so I added :

```haxe
@:native("EventEmitter")
extern class EventEmitter {
```

in the externs but it doesnt work either. Have you an idea about the issue ?

Last question about TextStyle: 

```haxe
// generate an error (in FlashDevelop)
var lTxt2:Text = new Text("pixiv4", { fontSize: 100, fontFamily: "MyOtherFont", fill: 0x000000, align: "left" } ); 

// works fine
var lStyle:TextStyleObject = { fontSize: 100, fontFamily: "MyOtherFont", fill: 0x000000, align: "left" };
var lTxt2:Text = new Text("pixiv4", lStyle );
```

Any other idea ? 

Thank you for your help.

**Comment by adireddy** [comment]
Yes, pixi 4 exposes event emitter.

`PIXI.utils.EventEmitter`

Just commited it to dev.

**Comment by adireddy** [comment]
released v4.0.1
